README.md

```diff
@@ -1,7 +1,6 @@
-# Python Utility Bot
+# Python Utility Bot
 
 [![Build Status](https://dev.azure.com/python-discord/Python%20Discord/_apis/build/status/Bot%20(Mainline))](https://dev.azure.com/python-discord/Python%20Discord/_build/latest?definitionId=1)
 [![Discord](https://discordapp.com/api/guilds/267624335836053506/embed.png)](https://discord.gg/2B963hn)
 
-This project is a Discord bot specifically for use with the Python Discord server. It provides numerous utilities
-and other tools to help keep the server running like a well-oiled machine.
+This project is a Discord bot specifically for use with the [Python Discord server](https://pythondiscord.com/). It provides numerous utilities and other tools to help keep the server running like a well-oiled machine.
```

Comment:
Splitting across multiple lines is normal for markdown to keep the line lengths from going out of control.

Reply:
Done.